packages/block-library/src/gallery/edit.js

```diff
@@ -347,9 +356,10 @@ class GalleryEdit extends Component {
 			images,
 			linkTo,
 			sizeSlug,
+			ids,
 		} = attributes;
 
-		const hasImages = !! images.length;
+		const hasImages = !! ids.length;
```

Comment:
can't currently get the list of images  from InnerBlocks using attribute selectors, so using the ids attribute for now